src/core/render/embed.js

```diff
@@ -117,17 +117,27 @@ export function prerenderEmbed({ compiler, raw = '', fetch }, done) {
     }
   });
 
-  let moveIndex = 0;
+  // keep track of which tokens have been embedded so far
+  // so that we know where to insert the embedded tokens as they
+  // are returned
+  const moves = [];
```

Comment:
```suggestion
  let moves = [];
```

Reply:
Can I ask why you're recommending this change?  Is there a style guide or anything that you're using to recommend this?

The recommended change is counter to my understanding of `const`/`let` in Javascript.  `const` ensures that a variable cannot be reassigned, however, it does not prevent modification objects/arrays.

In the code, even though the `moves` array has values inserted into it, it is never re-assigned.  Usage of `const` in this case prevents any accidental re-assignment of `moves` that wasn't intended.

If there's a style guide I should be following, I'm happy to change it, but on the surface, the requested change seems confusing and could potentially lead to an error in the future.